### PR TITLE
Cleanup README.md and transfer.md with latest kpod

### PR DESCRIPTION
**Pull request by rhatdan**
Add a couple of new ascicinema also

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@mrunalp @umohnani8 PTAL

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [![Build Status](https://img.shields.io/travis/kubernetes-incubator/cri-o.svg?maxAge=2592000&style=flat-square)](https://travis-ci.org/kubernetes-incubator/cri-o)
 [![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes-incubator/cri-o?style=flat-square)](https://goreportcard.com/report/github.com/kubernetes-incubator/cri-o)
 
-### Status: beta
+### Status: 1.0.0-rc1
```
Comment (mrunalp) [inline README.md]: nit: This need not match the entire version. We can just have `rc` or `release candidate`.

**Comment by TomSweeneyRedHat** [comment]
LGMT if @mrunalp is happy with the version

**Comment by umohnani8** [comment]
LGTM

**Comment by rhatdan** [comment]
@mrunalp Changed it to 
Release Candidate 1
PTAL

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**